x/auth/module.go

```diff
@@ -195,9 +195,14 @@ func ProvideModuleBasic() runtime.AppModuleBasicWrapper {
 type AuthInputs struct {
 	depinject.In
 
-	Config *modulev1.Module
-	Key    *store.KVStoreKey
-	Cdc    codec.Codec
+	Config    *modulev1.Module
+	ModuleKey depinject.OwnModuleKey
+	Key       *store.KVStoreKey
+	Cdc       codec.Codec
+
+	Authority               map[string]sdk.AccAddress     `optional:"true"`
```

Comment:
I'm confused as to how this works. Why can't we do this as an actual module config parameter in `module.proto`?

Reply:
This is what we do for every other modules.
The idea was not to have to import auth in every module requiring an authority:
https://github.com/cosmos/cosmos-sdk/blob/306a9a7/simapp/app.go#L204-L209

Wdyt?